dev-utils/docker-compose.yml

```diff
@@ -132,6 +132,7 @@ services:
       - SAUCE_USERNAME=${SAUCE_USERNAME}
       - SAUCE_ACCESS_KEY=${SAUCE_ACCESS_KEY}
       - MODE=${MODE}
+      - GOAL=${COVERAGE}
```

Comment:
the GOAL env var is not set but it works because we have an env var defined in the host that it is replaced on the docker-compose before launch it.

```
[2019-07-12T11:56:10.150Z] node-puppeteer         | + id
[2019-07-12T11:56:10.150Z] node-puppeteer         | uid=0(root) gid=0(root) groups=0(root)
[2019-07-12T11:56:10.150Z] node-puppeteer         | + node --version
[2019-07-12T11:56:10.150Z] node-puppeteer         | v8.16.0
[2019-07-12T11:56:10.150Z] node-puppeteer         | + npm --version
[2019-07-12T11:56:10.414Z] node-puppeteer         | 6.4.1
[2019-07-12T11:56:10.414Z] node-puppeteer         | + npx --version
[2019-07-12T11:56:10.414Z] node-puppeteer         | 6.4.1
[2019-07-12T11:56:10.414Z] node-puppeteer         | + cd /app
[2019-07-12T11:56:10.415Z] node-puppeteer         | + export
[2019-07-12T11:56:10.415Z] node-puppeteer         | + npm install
[2019-07-12T11:56:10.415Z] node-puppeteer         | declare -x APM_SERVER_URL="http://apm-server:8200"
[2019-07-12T11:56:10.415Z] node-puppeteer         | declare -x BRANCH_NAME="PR-322"
[2019-07-12T11:56:10.415Z] node-puppeteer         | declare -x BUILD_NUMBER="4"
[2019-07-12T11:56:10.415Z] node-puppeteer         | declare -x GOAL=""
```

Reply:
It seems now fixed

```
[2019-07-12T13:22:34.835Z] node-puppeteer         | + npm install
[2019-07-12T13:22:34.835Z] node-puppeteer         | declare -x APM_SERVER_URL="http://apm-server:8200"
[2019-07-12T13:22:34.835Z] node-puppeteer         | declare -x BRANCH_NAME="PR-322"
[2019-07-12T13:22:34.835Z] node-puppeteer         | declare -x BUILD_NUMBER="5"
[2019-07-12T13:22:34.835Z] node-puppeteer         | declare -x GOAL="coverage"
```